static/app/views/starfish/modules/APIModule/APIModuleView.tsx

```diff
@@ -0,0 +1,66 @@
+import {useQuery} from '@tanstack/react-query';
+import {Location} from 'history';
+
+import GridEditable, {GridColumnHeader} from 'sentry/components/gridEditable';
+
+const HOST = 'http://localhost:8000';
```

Comment:
```suggestion
const HOST = 'http://localhost:8080';
```
I changed the default to 8080 btw